Home.md

```diff
@@ -0,0 +1,141 @@
+Welcome to the ani-cli wiki!
+
+Ani-cli is a cli tool that scrapes Gogoanime and play's the anime locally on MPV(default) or VLC.
```

Comment:
ani-cli is a cli tool that scrapes Gogoanime and plays anime locally using MPV (default) or VLC.